content/software-security/comics/fighting-vulnerabilities/index.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "#1 - Fighting Vulnerabilities"
+description: "Join Inky on their fight against software vulnerabilities!"
+lead: "Join Inky on their fight against software vulnerabilities!"
+type: "article"
+date: 2023-07-25T15:35:05+00:00
+lastmod: 2023-07-26T14:58:56+00:00
+contributors: ["Michelle McAveety"]
+draft: false
+tags: ["Comic", "CVE"]
+images: []
+menu:
+  docs:
+    parent: "comics"
+weight: 001
+toc: true
+---
+
+![Comic featuring Chainguard's mascot, Inky the octopus, using their tentacles to defend a vulnerable computer from software vulnerabilities. The vulnerabilities take the form of robotic octopi charging forward as an army toward the computer.](seacuring-software-ep1.png)
```

Comment:
😆 https://www.dailymotion.com/video/x2voh0q

```suggestion
![Comic featuring Chainguard's mascot, Inky the octopus, using their tentacles to defend a vulnerable computer from software vulnerabilities. The vulnerabilities take the form of robotic octopuses charging forward as an army toward the computer.](seacuring-software-ep1.png)
```

Reply:
This is the best way I have ever been spell-checked lol

Reply:
Fwiw I don't think any plural is truly 100% correct, but I am fond of "octopodes" 🐙